app/views/teams/show.html.erb

```diff
@@ -1,8 +1,21 @@
+<script type='text/javascript'>
```

Comment:
Consider collecting scripts in the `app/javascripts` directory instead for inserting code snippets in the template. The bigger the application gets, the more confusing it will be to search for javascript code. [Here](http://railsapps.github.io/rails-javascript-include-external.html) you can find some details on how to handle javascripts in rails applications. See **Page-Specific Scripts**. Probably also an issue to discuss on the round table :v:

Reply:
Tried moving the script. Still works but in the console I now get this error:
![image](https://user-images.githubusercontent.com/17072869/33832329-0de0e8c6-de7c-11e7-8af5-c69c994d8500.png)

Reply:
I tried to do it just like the events people did in the events_new show page. When creating an event i get the sam error. ( Still just in the console but idk...)
![image](https://user-images.githubusercontent.com/17072869/33832507-b35d8a2a-de7c-11e7-9d8a-51518a606436.png)

Reply:
Okay, interesting! But your javascript code still works after migrating to the new location? Tests are all green? Then please commit and push, so that we can reproduce the issue. :heart_eyes: